server/src/assets/game/SignalToFlag.js

```diff
@@ -0,0 +1,254 @@
+const signalToFlagFull = {
+    "signal4": {
+        "fillColor": "black",
+        "backgroundColor": "white",
+        "cssClassForColor": "bg-white-black",
+        "number": "1",
+        "numberClass": "one-face"
```

Comment:
Should just be a single string that lists the visibility of each of the six dots, the circle colour, border colour, and background colour.